tests/plotting/test_widgets.py

```diff
@@ -164,6 +164,7 @@ def test_widget_slider(uniform):
     p.add_mesh_isovalue(uniform)
     p.close()
 
+    func = lambda value: value  # Does nothing
```

Comment:
Why?

Reply:
We have to change the callback from accepting `value, widget` to just `value` since `pass_widget=False` for the remainder of the test.